packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-credit-button-strategy.ts

```diff
@@ -98,6 +124,186 @@ export default class PaypalCommerceCreditButtonStrategy implements CheckoutButto
         }
     }
 
+    private async _onCancel() {
+        const state = this._store.getState();
+        const billingAddress = state.billingAddress.getBillingAddressOrThrow();
+        if (billingAddress) {
```

Comment:
This line can be removed due to the logic on previous line

Reply:
removed